guides/index.rst

```diff
@@ -1,16 +0,0 @@
-.. _navody:
-
-Návody
-======
-
-Návody jsou komunitní projekt `českých Pythonistů <https://python.cz>`__. Jejich cílem je poskytnout materiály a *know-how* pro pořadatele a částečně i účastníky (např. kouče) různých akcí, jako jsou srazy, workshopy, kurzy.
```

Comment:
I don't know if I want this text to completely disappear. Could you put it as a second paragraph into ` README.rst`, between "V dokumentaci..." and "Dokumentace je tvořena"?